James-Lycett/workplace-wellness
# Some big changes

Big pull request. I split the AdminHome page into 3 different components, added components for a table of a user's entries log, and shuffled the file system around a whole bunch. The user activity table ("UserRecordsTable.jsx" et al) still wants functionality for its buttons and probably some styling plus dates would be nice... I mostly just cloned the EmployeesList stuff over and adapted it all for entries instead. It still needs work but it's a step in the right direction.

I reorganized the file system so all the stuff we've been working on is under /Dashboard now instead of /AdminHome and slotted into neat little subdirectories for each component. Otherwise that /AdminHome folder was gonna have like 40 files in it to sort through. Just be warned the file structure is gonna look a lot different than what you've been seeing but I think it's a helpful move, lmk if you hate it.

To test:
- I messed with flowbite stuff, "npm i" might be necessary
- Check out the new file structure and the way I changed Dashboard.jsx (formerly AdminHome.jsx)
- Go to /admin/2/home or some other base userId at that route and check out the user table

Changes:
- Got the little logo working on the sidebar although it'll need adjusting, it pushed some text over
- Fixed login page popover issue where I had to refresh the page to get it to work
- AdminHome:
   > Renamed to Dashboard.jsx
   > Removed a bunch of unused junk
   > Split existing content into separate components
   > Added new component for !admin page view - "UserRecordsTable" to display entries table, will need !admin components 
      for !admin sidebar and progress charts, right now it's just the same stuff copied over, aside from UserRecordsTable
